source/guides/clear/mixer.rst

```diff
@@ -287,9 +313,58 @@ set to get a smaller kernel image, which will also be faster to load.
    Look in ~/mixer/update/www/<mix version> to see the update content in your
    workspace.
 
-#. Configure image. Edit the ister configuration file for your image to include
-   all of the bundles you want preinstalled in the image. If this is the first
-   time creating an image, first get a copy of the
+Example 3: Create an update for your mix
+========================================
+
+Next let’s create a new version of the mix. We’ll add a new package.
+
+#. Make the next mix. Create a new version of your mix, for the live image to
+   update to. Increment your mix version by 10:
+
+   .. code-block:: bash
+
+      mixer versions update
+
+#. Add the upstream :command:`curl` bundle to version 20 of the mix:
+
+   .. code-block:: bash
+
+      mixer bundle add curl
+
+#. Build your next mix version that incorporates the new bundle.
+
+   .. code-block:: bash
+
+      mixer build bundles
+      mixer build update
+
+Build delta-packs
```

Comment:
Please add "(optional)" after "Build delta-packs".
I also think this should be a sub bullet under Example 3.  So it should be Step #4.